src/schemas.py

```diff
@@ -1,31 +1,14 @@
-import json
 from datetime import datetime
-from enum import Enum
 from typing import Optional
+import json
 
-from .models import CustomModel
-
-
-class Product(Enum):
-    OTHER = 0
-    MOBILE_IOS = 1
-    MOBILE_ANDROID = 2
-    MOBILE_BACKEND = 3
-    PORTAL = 4
-    PCR = 5
-    PDP = 6
-    PCA = 7
-    PCP = 8
-    OHQ = 9
-    CLUBS = 10
-
-    def __str__(self):
-        return self.name
+from src.models import CustomModel, Product, RedisEvent
 
 
-class RedisEvent(CustomModel):
-    key: bytes | str
-    value: bytes | str
+class AuthRequest:
+    client_id: str
+    secret: str
+    product: Product = Product.OTHER
```

Comment:
I think we can remove this. Don't think we need to store these items on the analytics server